# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/rust_cascade/Cargo.toml
# Commit: b2e8af2fabb0
# Full Hash: b2e8af2fabb053ac02070415e9014a8a5e356d31
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-03-10 17:23:56
# Description:
#   Bug 1615974 - avoid memmapping CRLite filters in cert_storage r=jschanck,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D140266
# ==============================================================================

diff -r 902ab4b1756d -r b2e8af2fabb0 third_party/rust/rust_cascade/Cargo.toml
--- a/third_party/rust/rust_cascade/Cargo.toml	Wed Mar 09 22:35:35 2022 +0000
+++ b/third_party/rust/rust_cascade/Cargo.toml	Wed Mar 09 22:46:15 2022 +0000
@@ -3,18 +3,17 @@
 # When uploading crates to the registry Cargo will automatically
 # "normalize" Cargo.toml files for maximal compatibility
 # with all versions of Cargo and also rewrite `path` dependencies
-# to registry (e.g., crates.io) dependencies
+# to registry (e.g., crates.io) dependencies.
 #
-# If you believe there's an error in this file please file an
-# issue against the rust-lang/cargo repository. If you're
-# editing this file be aware that the upstream Cargo.toml
-# will likely look very different (and much more reasonable)
+# If you are reading this file be aware that the original Cargo.toml
+# will likely look very different (and much more reasonable).
+# See Cargo.toml.orig for the original contents.
 
 [package]
 name = "rust_cascade"
-version = "0.6.0"
+version = "1.2.0"
 authors = ["Mark Goodwin <mgoodwin@mozilla.com>", "Dana Keeler <dkeeler@mozilla.com>", "J.C. Jones <jc@mozilla.com>"]
-description = "A simple mmh3 based bloom filter cascade implementation in Rust."
+description = "A simple bloom filter cascade implementation in Rust."
 homepage = "https://github.com/mozilla/rust-cascade"
 documentation = "https://docs.rs/rust_cascade/"
 license = "MPL-2.0"